src/backend/storeManagers/hyperplay/utils.ts

```diff
@@ -342,3 +348,39 @@ export async function getEpicListingUrl(projectId: string): Promise<string> {
     return ''
   }
 }
+
+export const runModPatcher = async (appName: string) => {
+  // game_folder/client-patcher patch.exe -m patch/manifest.json
```

Comment:
do we need this comment?

Reply:
No, was just to keep track of the command to run from the CLI